clang/lib/CIR/CodeGen/CIRGenFunction.cpp

```diff
@@ -201,6 +201,20 @@ bool CIRGenFunction::constantFoldsToSimpleInteger(const Expr *cond,
   return true;
 }
 
+/// If the specified expression does not fold
+/// to a constant, or if it does but contains a label, return false.  If it
+/// constant folds return true and set the boolean result in `resultBool`.
+bool CIRGenFunction::constantFoldsToSimpleInteger(const Expr *cond,
```

Comment:
I believe we already have this function, but it was renamed to `constantFoldsToBool`